python/cugraph/tests/dask/opg_degree_testing.py

```diff
@@ -12,8 +12,9 @@
 def test_dask_opg_degree():
 
     gc.collect()
-    cluster = LocalCUDACluster(protocol="tcp", scheduler_port=0)
+    cluster = LocalCUDACluster()
     client = Client(cluster)
+    Comms.initialize()
```

Comment:
Is `Comms` the best name we can think of here? 
The handle stores CUDA libs context and device properties in addition to the communicator. It is also quite specific to cuGraph. I think the name should capture all this so be a bit more self-documented.

Reply:
It is a cugraph encapsulation of raft Comms. It does contain the handle which in turn stores much more but the thought behind naming it Comms is the fact that it is only accessed and initialized when a communicator is needed for multi-gpu. Else we internally use default_handle in single gpu functionalities and this import is not required. However, I do see the point you put forth. We could call it environment initialization instead of communicator but that could be confusing.

Reply:
I see. I think it is ok then, we can just carefully select the  `import cugraph.comms.comms as ...` in notebooks for instance.